hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -108,15 +108,69 @@ Lastly, while the different partitions may be identical in some attributes, they
 
 For default assumptions of the design, see the [default configuration]({{< relref "#flash-default-configuration" >}}).
 
+#### Addresses Map
+
+##### Bank Address
+The flash address map is built upon the bank base address.
+The bank size is based upon the number of pages in the data partition.
+The first bank's address is always `0x0`.
+The second bank's address is `0x0 + size_of_bank_in_bytes`.
+
+For example:
+Assume each bank is 512KB in size.
+The address of bank 0 is `0x0`.
+The address of bank 1 is `0x80000`
+
+##### Page Address
+The address of a particular page is calculated based on the page size and the index number of the page.
+
+For example:
+Assume each page is 2KB in size.
```

Comment:
But line 59 suggests 1KB pages... Could you clarify?

Reply:
gasp! did i do my math wrong? let me check.

Reply:
thank you! phew, the documentation was out of sync.